ReactWindows/ReactNative/Views/TextInput/ReactPasswordBoxManager.cs

```diff
@@ -527,6 +525,19 @@ private void OnKeyDown(object sender, KeyRoutedEventArgs e)
                             textBox.GetTag(),
                             textBox.Password));
             }
+
+            if (!e.Handled)
```

Comment:
>e.Handled [](start = 17, length = 9)

Can this be true in cases besides when `e.Key == VirtualKey.Enter`?

Reply:
Not in this case, but I followed the pattern in other places as well and I think it's a good one. If we handle the event locally (like in this case Enter sends this alternate "InputSubmit" event), we stop sending the other lower level "KeyDown")
(this makes me wonder why the WPF version checks against Enter rather than Return...)

Reply:
Yes, that's code that was there. I'll test a TextInput in a WPF app to see what happens.
But the more I see stuff like this the more I'm concerned it depends on OS, SDK, Target, you name it. I may just add a "||" on the WPF side.

Reply:
Ok, I think I discovered the issue with Space/Enter. They're both defined in WPF:
        Return = 6,
        Enter = 6,
The GetKeyConstants in KeyHelpers removes the duplicates, so in our case Enter suffers.
I know the "GetValues" based trick to obtain the list is fragile when containing duplicates. But I think there's a better way that works with dupes (I remember using it before, I'll dig). If this succeeds, there's no need to use "Return" anymore.